### ISSUE TITLE
Add mime types for .sass and .scss

### ISSUE THREAD

**Issue by jecisc** [SeasideSt/Seaside#927]
If I want to import a sass file it is imported as a binary file. The problem seems to comes from that fact that there is no mime type for .sass or .scss files. 

I don't know anything about mimetypes but looking on internet it looks like we could add

```
'sass' 'text/x-sass'
'scss' 'text/x-scss'
```

To `WAAbstractFileLibrary>>defaultMimeTypes`

**Comment by jecisc** [comment]
Also, currently batch file are imported as binary files and not text. Maybe they should be imported as text.

**Comment by marschall** [comment]
Do you actually serve .sass .scss and .bat files from your web application?

**Comment by jecisc** [comment]
I have project that does not rely on ngnix/apache. To manage the sass/scss files I have scripts to export and import the sass/scss and keep them in a FileLibrary.

When we have to do small fixes it is good to be able to do these changes directly from Pharo without exporting everything.

**Comment by jecisc** [comment]
It is also good for code review from github's interface of Monticello's tools.

**Comment by jecisc** [comment]
And a last argument is that it is easier to merge in case of conflicts

**Comment by marschall** [comment]
Yes, if you have them in a file library it's better to have them in source. It's also smaller.

**Comment by theseion** [comment]
I have no opinion on this, other than that I don't like adding MIME types for every possible file type (standard types are one thing but everbody can invent their own `x-` types).

**Comment by jecisc** [comment]
Do you think that we should add a way to read some files with some extensions by default as non binary file and let the user override a method of the FileLibrary to define those extensions instead?

**Comment by marschall** [comment]
We already have a lot of non-standard file types

```st
WAAbstractFileLibrary  defaultMimeTypes count: [ :each |
	each includesSubstring: '/x-' ]
```

Answers 264 in my image.

**Comment by marschall** [comment]
Fixed by #961
